modules/_node-scripts/package.json

```diff
@@ -34,6 +34,7 @@
 		"jest-environment-jsdom-thirteen": "1.0.1",
 		"jest-fetch-mock": "^3.0.3",
 		"liferay-theme-tasks": "11.5.1",
+		"micromatch": "4.0.2",
```

Comment:
This is the same matching package that fast-glob uses, so we aren't actually expanding our dependency footprint here